plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -1202,7 +1209,42 @@ triggers:
         description: Alert
         type: alert
         required: false
-
+  get_query_results:
+    title: Get Query Results
+    description: Runs the XQL and returns the output data results
+    input:
+      frequency:
+        title: Frequency
+        description: Poll frequency in seconds
+        type: integer
+        required: true
+        default: 5
+        example: 5
+      query:
+        title: Query
+        description: String of the XQL query
+        type: string
+        required: true
+        example: dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3
+      tenants:
+        title: Tenants
+        description: String that represents additional information regarding the action
```

Comment:
I know tenants is a value that exists elsewhere, but is there a more informative description we can add?